torch/_inductor/ir.py

```diff
@@ -6660,19 +6687,22 @@ def __init__(self, layout, inputs, constant_args, reduce_op):
     def get_mutation_names(self):
         return [self.inputs[0].get_name()]
 
+    def get_unbacked_symbol_defs(self):
+        return {}
+
```

Comment:
Is this used anywhere?  I don't see it being called.

Reply:
https://github.com/pytorch/pytorch/blob/3be0e1cd587ece8fa54a3a4da8ae68225b9cbb9b/torch/_inductor/scheduler.py#L1342

If I don't define it, this call will fail.